src/main/java/lotto/domain/BonusNumber.java

```diff
@@ -0,0 +1,20 @@
+package lotto.domain;
+
+import static lotto.util.LottoUtil.verifyRangeOfLottoNumber;
+
+public class BonusNumber {
+    private final int bonusNumber;
+
+    public BonusNumber(int bonusNumber) {
+        this.bonusNumber = validateBonusNumber(bonusNumber);
+    }
+
+    private static int validateBonusNumber(int number) {
+        verifyRangeOfLottoNumber(number);
+        return number;
+    }
```

Comment:
return을 꼭 할 필요가 있을까요? 
```suggestion
    public BonusNumber(int bonusNumber) {
        validateBonusNumber(bonusNumber);
        this.bonusNumber = bonusNumber;
    }

    private static void validateBonusNumber(int number) {
        verifyRangeOfLottoNumber(number);
    }
```
이런 코드는 어떻게 생각하시나요!

Reply:
validate가 유틸성이 아니라 리턴해도 괜찮다고 생각했습니다!